test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -249,7 +249,7 @@ public void NgramWorkout()
                 .Read(sentimentDataPath);
 
             var est = new WordTokenizingEstimator(Env, "text", "text")
-                .Append(new ValueToKeyMappingEstimator(Env, "text", "terms"))
+                .Append(new ValueToKeyMappingEstimator(Env, "terms", "text"))
                 .Append(new NgramExtractingEstimator(Env, "terms", "ngrams"))
```

Comment:
>NgramExtractingEstimator [](start = 28, length = 24)

Hmmm. This didn't get fixed?

Reply:
I am not converting the NGramExtracting Estimator on this PR. It will come in the next PR. 

---
In reply to: [248095484](https://github.com/dotnet/machinelearning/pull/2118#discussion_r248095484) [](ancestors = 248095484)